apps/web/src/components/NavMenusSection/data.tsx

```diff
@@ -0,0 +1,102 @@
+import type { ReactNode } from "react";
+import {
+  BookOpenIcon,
+  ChartBarIcon,
+  CommandLineIcon,
+  Squares2X2Icon,
+} from "@heroicons/react/24/solid";
+
+import { env } from "~/env.mjs";
+import EthereumIcon from "~/icons/ethereum.svg";
+import {
+  buildBlobStatsRoute,
+  buildBlockStatsRoute,
+  buildBlocksRoute,
+  buildTransactionStatsRoute,
+  buildTransactionsRoute,
+  buildBlobsRoute,
+} from "~/utils";
+
+function resolveApiUrl(): string {
+  if (env.NEXT_PUBLIC_NETWORK_NAME === "mainnet") {
+    return "https://api.blobscan.com";
+  }
+
+  return `https://api.${env.NEXT_PUBLIC_NETWORK_NAME}.blobscan.com`;
+}
+
+export type MenuItem = {
+  label: string;
+  href: string;
+  icon: ReactNode;
+  type: "single";
```

Comment:
No need to have a `type` field. Let's use [type guards](https://www.typescriptlang.org/docs/handbook/advanced-types.html) for each type instead

Reply:
We can export these functions and use them to determine if an object belongs to one of these types.

Reply:
Done 2f1289c